rcaracaus/pledge-vegan-node
# Caught errors which caused server to crash on certain urls

Steps to test:

1. Go to localhost:3000
2. Go to any url and expect the server to respond with either the page or a 404 (ie. localhost:3000/s gives you a 404)
3. Check that every valid sub url also handles their sub urls correctly and responds with either a page or a 404 (ie. localhost:3000/vegans/micheal-bubble gives you a 404)
4. Make sure all original functionality has been kept.
